tests/imgcat

```diff
@@ -25,6 +25,9 @@ function print_st() {
 #   inline: 0 or 1
 #   base64contents: Base64-encoded contents
 function print_image() {
+    if [[ -z "$1" ]]; then
+      exit 0
```

Comment:
I think this is wrong. The reason you got a broken image is the call on line 105. Adding this would break `cat image.png | imgcat`. A better fix would be to not accept input from stdin if there is a filename command-line argument.